source/anti-patterns/language.md

```diff
@@ -31,7 +31,7 @@ There are a couple of pitfalls:
 - combining with overriding logic such as `overrideAttrs` function in nixpkgs has a surprising behaviour
   of not overriding every reference.
 
-A better way is to use simpler `let .. in`:
+A better way is to use simpler but more verbose `let .. in`:
```

Comment:
```suggestion
A better way is to use `let ... in`:
```

I'm not sure in what sense `let ... in` is simpler, and I don't think it's worthwhile pointing out it requires more characters to express.